lib/camt_parser/053/statement.rb

```diff
@@ -39,7 +39,7 @@ def electronic_sequence_number
 
       def opening_balance
         @opening_balance ||= begin
-          bal = @xml_data.xpath('Bal/Tp//Cd[contains(text(), "PRCD")]').first.ancestors('Bal')
+          bal = @xml_data.xpath('Bal/Tp//Cd[contains(text(), "OPBD") or contains(text(), "PRCD")]').first.ancestors('Bal')
```

Comment:
Type may only have one code child. The order does not really matter, but I can change it if you prefer?